VideoLocker/src/main/java/org/edx/mobile/model/Page.java

```diff
@@ -0,0 +1,45 @@
+package org.edx.mobile.model;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+import java.util.List;
+
+public class Page<T> {
```

Comment:
This class should also have the definition of `PAGE_SIZE` as it is being defined explicitly in each implementation.